CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift

```diff
@@ -269,20 +271,26 @@ public final class Camera2ViewController: UIViewController, CameraScreen {
         qrCodeOverLay.viewWillDisappear()
     }
 
-    private func showUploadButton() {
+    private func configureUploadButton() {
         if giniConfiguration.fileImportSupportedTypes != .none {
             cameraPane.fileUploadButton.isHidden = false
         } else {
             cameraPane.fileUploadButton.isHidden = true
+            cameraButtonsViewModel.importAction = { [weak self] in
```

Comment:
[Clarification] @zladzeyka  It seems that import action won't be set when the fileUploadButton isn't hidden?

Reply:
Thanks, @chriskryn a very good catch!